src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/ResultsGroup.css

```
@@ -0,0 +1,22 @@
+.results-group-title-button {
+    flex-direction: row-reverse !important;
+    gap: 2px !important;
+    padding-inline-start: 0 !important;
+}
+
+.results-group-title-container {
+    display: flex;
+    flex-grow: 1;
+}
+
+.results-group-title-text-container {
+    flex-grow: 1;
+    gap: 0.1rem;
```

with the Inter font change, how about
```suggestion
    gap: 0.2rem;
```